public/javascript/vat-v1.js

```diff
@@ -1,4 +1,22 @@
 $(document).ready(function() {
   //initialise <details> polyfill from frontend toolkit
   GOVUK.details.init();
-});
\ No newline at end of file
+});
```

Comment:
This file needs to be `vat-v2.js` to match `govuk_wrapper`... think this is a bug